spotbugs/src/doc/FilterFile.txt

```diff
@@ -1,125 +1 @@
-   "String".)  Note that "params" and "returns" are optional; you can
-   just specify "name", and the clause will match all methods with
-   that name.  However, if you specify either "params" or "returns",
-   you must specify both of them.
-
-   <Or> combines Match clauses as disjuncts.  I.e., you can put two
-   "Method" elements in an Or clause in order match either method.
-
-========
-Caveats:
-========
-
-Match clauses can only match information that is actually contained in the
-bug instances.  Every bug instance has a class, so in general, excluding
-bugs by class will work.
-
-Some bug instances have two classes.  For example, the DE (dropped exception)
-bugs report both the class containing the method where the dropped exception
-happens, and the class which represents the type of the dropped exception.
-Only the FIRST (primary) class is matched against Match clauses.
-So, for example, if you want to suppress IC (initialization circularity)
-reports for classes "com.foobar.A" and "com.foobar.B", you would use
-two Match clauses:
-
-   <Match class="com.foobar.A">
-      <BugCode name="IC" />
-   </Match>
-
-   <Match class="com.foobar.B">
-      <BugCode name="IC" />
-   </Match>
-
-Many kinds of bugs report what method they occur in.  For those bug instances,
-you can put Method clauses in the Match element and they should work
-as expected.
-
-=========
-Examples:
-=========
-
-  1. Match all bug reports for a class.
-
-     <Match class="com.foobar.MyClass" />
-
-  2. Match certain tests from a class.
-     <Match class="com.foobar.MyClass">
-       <BugCode name="DE,UrF,SIC" />
-     </Match>
-
-  3. Match certain tests from all classes.
-
-     <Match classregex=".*" >
-       <BugCode name="DE,UrF,SIC" />
-     </Match>
-
-  4. Match bug types from specified methods of a class.
-
-     <Match class="com.foobar.MyClass">
-       <Or>
-         <Method name="frob" params="int,java.lang.String" returns="void" />
-         <Method name="blat" params="" returns="boolean" />
-       </Or>
-       <BugCode name="DC" />
-     </Match>
-
-=================
-Complete Example:
-=================
-
-<FindBugsFilter>
-     <Match class="com.foobar.ClassNotToBeAnalyzed" />
-
-     <Match class="com.foobar.ClassWithSomeBugsMatched">
-       <BugCode name="DE,UrF,SIC" />
-     </Match>
-
-     <!-- Match all XYZ violations. -->
-     <Match classregex=".*" >
-       <BugCode name="XYZ" />
-     </Match>
-
-     <!-- Match all doublecheck violations in these methods of "AnotherClass". -->
-     <Match class="com.foobar.AnotherClass">
-       <Or>
-         <Method name="nonOverloadedMethod" />
-         <Method name="frob" params="int,java.lang.String" returns="void" />
-         <Method name="blat" params="" returns="boolean" />
-       </Or>
-       <BugCode name="DC" />
-     </Match>
-</FindBugsFilter>
+Refer http://spotbugs.readthedocs.io/en/latest/filter.html instead of this page.
```

Comment:
Why should we keep this file?
If we want to keep it, may be add that "the filter.html is generated from docs/filter.rst"?